src/signTx.c

```diff
@@ -338,20 +338,11 @@ static void signTx_handleInit_ui_runStep()
 		        "New script transaction" :
```

Comment:
Not for this PR, but when renaming the singing mode SCRIPT -> MULTISIG, we want to change this string --- please note it somewhere so that we do not forget. If "New multisig transaction" is too long to fit on the display, I'd go with "Multisig transaction".

Reply:
Will try.